scripts/image-diff

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
```

Comment:
This functionality will probably be incorporated into the `wad2image`, but I'l make sure to include that to any script I add to Freedoom.